tethysext/atcore/templates/atcore/app_users/manage_resources.html

```diff
@@ -3,161 +3,163 @@
+                    {{ resource.debugging }}
+                  </td>
+                  {% endif %}
+
+                  {# BUTTONS #}
+                  <td class="btn-td">
+                    <div class="btn-toolbar pull-right">
+                      {% if resource.editable %}
+                        <a class="btn btn-default btn-edit btn-flat" href="{% url base_resource_href|add:'_edit_resource' resource.id %}" data-toggle="tooltip" data-placement="top" title="Edit"><span class="glyphicon glyphicon-edit"></span></a>
+                      {% endif %}
+                      {% if resource.deletable %}
+                        <a class="btn btn-default btn-delete-manage btn-delete btn-flat" href="javascript:void(0);" data-id="{{ resource.id }}" data-toggle="tooltip" data-placement="top" title="Delete"><span class="glyphicon glyphicon-trash"></span></a>
+                      {% endif %}
+                      {% if resource.action == 'processing' %}
+                        <a class="btn btn-s btn-warning btn-flat btn-action btn-resource-processing" href="{{ resource.action_href }}" data-toggle="tooltip" data-placement="top" title="{{ resource.action_title }}"><span class="glyphicon glyphicon-refresh"></span></a>
+                      {% elif resource.action == 'error' %}
+                        <a class="btn btn-s btn-error btn-flat btn-action btn-resource-error" href="{{ resource.action_href }}" data-toggle="tooltip" data-placement="top" title="{{ resource.action_title }}"><span class="glyphicon glyphicon-remove"></span></a>
+                      {% else %}
+                        <a class="btn btn-s btn-primary btn-flat btn-action btn-resource-launch" href="{{ resource.action_href }}" data-toggle="tooltip" data-placement="top" title="{{ resource.action_title }}"><span class="glyphicon glyphicon-chevron-right"></span></a>
+                      {% endif %}
+                    </div>
+                  </td>
+                </tr>
+              {% endfor %}
+            </tbody>
+          </table>
+        </div>
       </div>
-    </div>
-    {% include "atcore/pagination_template.html" with pagination_info=pagination_info where="footer" %}
-  {% else %}
+      {% include "atcore/pagination_template.html" with pagination_info=pagination_info where="footer" %}
+    {% else %}
       {% if show_new_button %}
       <div class="new-resource center-parent">
         <div class="centered">
-          <h4>No {{ type_plural }} Found</h4>
-          <a class="btn btn-primary btn-huge" href="{% url tethys_app.namespace|add:':app_users_new_resource' %}">Upload New {{ type_singular }}</a>
+          <h4>No {{ page_title }} Found</h4>
+          <a class="btn btn-primary btn-huge" href="{% url base_resource_href|add:'_new_resource' %}">Upload New {{ type_singular }}</a>
         </div>
       </div>
       {% else %}
       <div class="new-resource center-parent">
-        <h4 class="centered">No {{ type_plural }} Available</h4>
```

Comment:
Please change `page_title` back to `type_plural` where you changed it. These are meant to be used in places where the plural version of the resource display name should be used. Notice there is a usage of type_singular too. The fact that the `page_title` is the same as `type_plural` is coincidental for this view. They could be different if the user wants them to be. For example, they could override the ManageResources view and set page title to : "Manage Models" or something else. We wouldn't want the other text the change. For example: "No Manage Models Found"???